pkg/apitype/stacks.go

```diff
@@ -48,13 +48,18 @@ type CreateStackRequest struct {
 	// The rest of the StackIdentifier (repo, project) is in the URL.
 	StackName string `json:"stackName"`
 	// An optional set of tags to apply to the stack.
-	Tags map[StackTagName]string `json:"tags,omitEmpty"`
+	Tags map[StackTagName]string `json:"tags,omitempty"`
 }
 
 // CreateStackResponseByName is the response from a create Stack request.
 type CreateStackResponseByName struct {
 	// The name of the cloud used if the default was sent.
 	CloudName string `json:"cloudName"`
+	// The number of stacks owned by the user after this stack was created.
```

Comment:
Nit: We use "user" in the first comment, and "owner" in the second. I think we should be consistent in using "owner" to refer to where stacks go. (But maybe "organization" would be even better?)